Makefile

```diff
@@ -56,6 +56,19 @@ ifneq ($(TAG),latest)
 BUNDLE_VERSION ?= --version=$(shell sed s/^v// <<< $(TAG))
 endif
 
+# To get the GIT commit id
+GIT_COMMIT ?= $(shell git rev-list -1 HEAD)
+
+# Fetch the git tag
+GIT_TAG ?= $(shell git describe --tags HEAD=0 2>/dev/null)
```

Comment:
Thanks for checking with `git describe`. I get an error with `HEAD=0`, but `HEAD` works for me. If `git describe` is troublesome, using `git rev-list` as an alternative is ok too. Maybe this works?

```
GIT_TAG ?= $(shell git describe --tags HEAD 2>/dev/null || echo $(GIT_COMMIT))
```

Reply:
looks good. 
Although the version looks pretty big, if it's okay I can use short commit instead of using GIT_COMMIT

Reply:
great, thanks!

Reply:
updated